drivers/misc/optee.c

```diff
@@ -259,16 +314,14 @@ static int optee_from_msg_param(FAR struct tee_ioctl_param *params,
   return 0;
 }
 
-static int
-optee_ioctl_close_session(FAR struct optee_priv_data *priv,
-                          FAR struct tee_ioctl_close_session_arg *arg)
+static int optee_close_session(FAR struct optee_priv_data *priv,
+                               uint32_t session)
```

Comment:
merge to previous line. Let's move the new prototype function to previous patch to avoid the big change in this patch.